src/index.ts

```diff
@@ -1,21 +1,93 @@
+
+import {
+    INotebookTracker
+} from '@jupyterlab/notebook';
+
+import {
+  ContentsManager
+} from '@jupyterlab/services';
+
+import {
+  ICommandPalette
+} from '@jupyterlab/apputils';
+
 import {
   JupyterLab,
   JupyterLabPlugin
 } from '@jupyterlab/application';
 
+import {AllOfUsConfig} from './config';
+import {ConceptsWidget} from './concepts-widget';
+
 import '../style/index.css';
 
+// Activate the jupyterhub extension.
+function activateExtension(app: JupyterLab,
+                           notebooks: INotebookTracker,
+                           palette: ICommandPalette): void {
+    const tracker = notebooks;
+    let allOfUsConfig = new AllOfUsConfig();
+
+    // Create a single widget
+    let conceptsWidget: ConceptsWidget = new ConceptsWidget(allOfUsConfig);
```

Comment:
I'm not sure the type is necessary here, I'm pretty sure ts can infer it in this case.